### PR TITLE
Usage record enable config

### LINKED ISSUE #10: [Bug] Add support for
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

Several customers do not have some combination of the sources `address` or `usage_record`. It produces the below error since we use `dbt_utils.star()`:

```
Syntax error: SELECT list must not be empty
```

### How would you implement this feature?

`address` - Investigate if we can remove this altogether. It's [brought into the end model](https://github.com/fivetran/dbt_twilio/blob/71baaccc6f4e668e6cfed9dbe49b97df917b7f5e/models/twilio__message_enhanced.sql#L13-L17), but never actually used.

`usage_record` - [This](https://github.com/fivetran/dbt_twilio/blob/71baaccc6f4e668e6cfed9dbe49b97df917b7f5e/models/twilio__account_overview.sql#L48) would be easy to allow disabling, or we can leave as-is.

### Describe alternatives you've considered

_No response_

### Are you interested in contributing this feature?

- [ ] Yes.
- [ ] Yes, but I will need assistance.
- [ ] No.

### Anything else?

_No response_

### LINKED ISSUE #12: [Documentation] Move badges below H1 heading
*Copied from [fivetran/dbt_tiktok_ads_source #24](https://github.com/fivetran/dbt_tiktok_ads_source/issues/24).*

### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

We need to move the badges at the top of the README to be below the H1 heading. This is to be consistent with other popular README formats and also ensure there is no relevant information above the H1 heading. Additionally, we will need to adjust the orientation of the badges to be consistent with the rest of the README which is left-oriented.

### How would you implement this feature?

Move the badge information below the H1 heading and change the `<p align=` to be `"left"`.

### For Example
In the before changes codeblock below, the H1 header (identified by a single `#`) needs to be moved to the very top. Then we must ensure everything within the `<p>` tag is below the H1 header, but before the first H2 heading (identified by a double `##`).
> Note: This example uses TikTok Ads Source. These changes should be applied consistently across all relevant repositories, using the repository-specific badge information. 

#### Before Changes
```
<p align="center">
    <a alt="License"
        href="https://github.com/fivetran/dbt_tiktok_ads_source/blob/main/LICENSE">
        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
    <a alt="dbt-core">
        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.3.0_<2.0.0-orange.svg" /></a>
    <a alt="Maintained?">
        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
    <a alt="PRs">
        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
</p>

# Tiktok Ads Source dbt Package ([Docs](https://fivetran.github.io/dbt_tiktok_ads_source/))
## What does this dbt package do?
```

#### After Changes
Notice the `<p align="left">` update as well as the location of the badges between the H1 and H2 header with appropriate spacing.
```
# Tiktok Ads Source dbt Package ([Docs](https://fivetran.github.io/dbt_tiktok_ads_source/))

<p align="left">
    <a alt="License"
        href="https://github.com/fivetran/dbt_tiktok_ads_source/blob/main/LICENSE">
        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
    <a alt="dbt-core">
        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.3.0_<2.0.0-orange.svg" /></a>
    <a alt="Maintained?">
        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
    <a alt="PRs">
        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
</p>

## What does this dbt package do?
```

### Are you interested in contributing this feature?

- [ ] Yes.
- [ ] Yes, but I will need assistance.
- [ ] No.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:** https://github.com/fivetran/dbt_twilio_source/issues/10

**This PR will result in the following new package version:** v0.4.0

I made it breaking in the case if usage_record is disabled which would alter the schema.

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## Features
- Added the ability to disable models related to the `USAGE_RECORD` source table. Refer to the [README](https://github.com/fivetran/dbt_twilio_source?tab=readme-ov-file#step-4-enablingdisabling-models) for more details

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [ ] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Ensure that disabling the source table will remove the related staging model from populating.

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# dbt_twilio_source v0.4.0
+
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:2]: Please add the PR reference link here

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+## Features
+- Added the ability to disable models related to the `USAGE_RECORD` source table. Refer to the [README](https://github.com/fivetran/dbt_twilio_source?tab=readme-ov-file#step-4-enablingdisabling-models) for more details
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md]: Can you highlight the variable name here as well in addition to the link out to the README

File: README.md [hunk 3]
```diff
@@ -66,6 +66,7 @@ By default, all variables are assumed to be `true`.
 vars:
   using_twilio_call: False # Disable this if not using call
   using_twilio_messaging_service: False # Disable this if not using messaging_service
+  using_twilio_usage_record: False # Disable this if not using call
```
Comment (fivetran-joemarkiewicz) [inline README.md]: This needs to be updated to reference the appropriate source table for this variable.
```suggestion
  using_twilio_usage_record: False # Disable this if not using usage_record
```

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM with just two small change requests in the CHANGELOG and a small update in the README before release review, but no need for this to block initial approval.

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-reneeli) [reply CHANGELOG.md:2]: updated

File: CHANGELOG.md (hunk 2 above)
Comment (fivetran-reneeli) [reply CHANGELOG.md]: updated, though the README link should already be there

File: README.md (hunk 3 above)
Comment (fivetran-reneeli) [reply README.md]: thanks for catching! updated

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
+# dbt_twilio_source v0.4.0
+
+[PR #13](https://github.com/fivetran/dbt_twilio_source/pull/13) includes the following updates:
```
Comment (fivetran-avinash) [inline CHANGELOG.md:3]:
```suggestion
[PR #13](https://github.com/fivetran/dbt_twilio_source/pull/13) includes the following updates:

```
Adding spacing.

**Review by fivetran-avinash — APPROVED** [review]
one small comment but lgtm

**Comment by fivetran-reneeli** [comment]
Also added updates to address #12

**Pull request merged**